README.md

```diff
@@ -1,108 +1,28 @@
-  Then modify `src/main/resources/application.properties` file and save it.
-
-    ``` txt
-    azure.documentdb.uri=put-your-documentdb-uri-here
-    azure.documentdb.key=put-your-documentdb-key-here
-    azure.documentdb.database=put-your-documentdb-databasename-here
-    ``` 
-
-* If you don't want to modify configuration in the source code manually, you can put variables in this file and 
-  set their values in system environment variables: `DOCUMENTDB_URI`, `DOCUMENTDB_KEY` and `DOCUMENTDB_DBNAME`.
-  Then maven will substitute them during the build phase.
-    ``` txt
-    azure.documentdb.uri=@env.DOCUMENTDB_URI@
-    azure.documentdb.key=@env.DOCUMENTDB_KEY@
-    azure.documentdb.database=@env.DOCUMENTDB_DBNAME@
-    ``` 
-
-## Run it
-
-1. package the project using `mvn package`
-1. Run the project using `java -jar target/todo-app-java-on-azure-1.0-SNAPSHOT.jar`
-1. Open `http://localhost:8080` you can see the web pages to show the todo list app
-
-## Clean up
-
-Delete the Azure resources you created by running the following command:
-
-```bash
-az group delete -y --no-wait -n <your-resource-group-name>
-```
-
-## Contributing
-
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.microsoft.com.
-
-When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-
-## Useful link
-- [Azure Spring Boot Starters](https://github.com/Microsoft/azure-spring-boot)
-- [Azure Maven plugins](https://github.com/Microsoft/azure-maven-plugins)
+* Follow the steps for creating the Azure App Insights resource (as shown in the Requirements section) and copy the instrumentation key.
+  Paste the instrumentation key into the 'src/main/resources/ApplicationInsights.xml' file and save it.
+  
+  <!-- The key from the portal: -->
+      <InstrumentationKey>put-your-instrumentation-key-here</InstrumentationKey>
+ 
+* In addition, you may notice that the following has already been configured for you to setup App Insights:
+1. The dependency to Application Insights has been added to the pom.xml:
+   <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>applicationinsights-web</artifactId>
+             <version>[1.0,)</version>
```

Comment:
Please update version to 2.0.0-BETA for time being. The [1.0.,) will pull the last 1.x version which won't be the latest.

Reply:
Done.